### PR TITLE
fix-38380: New composer : insert youtube link .

### PR DESCRIPTION
**Pull request by IlhemEssaadi**
ISSUES : we moved the location of the video plugin from the news module to the social module so we changed in the code the path of this plugin.

### CODE REVIEW

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue [hunk 1]
```diff
@@ -537,7 +537,7 @@ export default {
         // Disable suggester on smart-phone landscape
         extraPlugins = 'simpleLink,selectImage';
       }
-      CKEDITOR.addCss('.cke_editable { font-size: 14pt; font-family: Helvetica, regular, sans-serif; }');
+      CKEDITOR.addCss('.cke_editable { font-size: 14pt; font-family: Helvetica, regular, sans-serif; } .video { width: 900px; height: 506px; margin: 0 auto; }');
```
Comment (rdenarie) [inline webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue:544]: should this be in less file ?

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue (hunk 1 above)
Comment (IlhemEssaadi) [reply webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue:544]: > should this be in less file ?
I want to change the height and width of the video that is displayed in ckeditor, that's why I did not add it in the less file I add them directly in ckeditor .

Comment (rdenarie) [reply webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue:544]: and why it couldnt be in a less file ?

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue (hunk 1 above)
Comment (rdenarie) [reply webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue:544]: Ok, this is a specific css for news

**Comment by rdenarie** [comment]
Can you resolv conflict in this PR ?

**Review by rdenarie — CHANGES_REQUESTED** [review]
there is conflict in this pr

**Review by rdenarie — APPROVED** [review]

**Pull request merged**